EIPS/eip-7581.md

```diff
@@ -0,0 +1,68 @@
+---
+eip: 7581
+title: Increase block gas target and gaslimit
+description: Increase default block gas by 50%
+author: Ben Adams (@benaadams)
+discussions-to: https://ethereum-magicians.org/t/increase-block-gas-target-and-gaslimit/17626
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-12-24
+requires: 4844, 7251
+---
+
+## Abstract
+
+Increase default block gas target by 50% from 15M to 22.5M and block gaslimit from 30M to 45M
+
+## Motivation
+
+It has been the longest period in Ethereum's history without a gasblock increase 2 years, 4 months since [EIP-1559](./eip-1559.md) went live. Hardware and diskspace has improved in this time as has execution client performance. L1 blockspace remains in demand and the network should support this modest increase.
+
+
+## Specification
+
+* Increase default block gaslimit from 30M to 45M
+* Target remains half of that increasing from 15M to 22.5M 
+
+## Rationale
+
+While increasing block speed could also be an option it may have issues due to assumptions that have crept in or larger consensus times (which [EIP-7251](./eip-7251.md) seeks to address to some degree)
+
+## Backwards Compatibility
+
+No backward compatibility issues found.
+
+## Security Considerations
+
+[EIP-1153](./eip-1153.md) `TSTORE` presents a new way to allocate memory on a node with linear cost. In other words, each TSTORE allows the developer to store 32 bytes for 100 gas, excluding any other required operations to prepare the stack. Given 45 million gas, the maximum amount of memory that can be allocated using TSTORE is:
```

Comment:
how is this a security consideration?

Reply:
Is increasing the amount of memory that can be allocated by a single transaction which could be a DoS vector if unreasonably large or EL clients not prepared for it. Though not sure it would be an issue as isn't that large in the scheme of things